instana/instrumentation/sqlalchemy.py

```diff
@@ -47,20 +49,27 @@ def receive_after_cursor_execute(**kw):
             if scope is not None:
                 scope.close()
 
-    @event.listens_for(Engine, 'dbapi_error', named=True)
-    def receive_dbapi_error(**kw):
-        context = kw['context']
 
-        if context is not None and hasattr(context, '_stan_scope'):
-            scope = context._stan_scope
+    error_event = "handle_error"
+    # Handle dbapi_error event; deprecated since version 0.9
+    if sqlalchemy.__version__[0] == "0":
+        error_event = "dbapi_error"
+
+
+    @event.listens_for(Engine, error_event, named=True)
+    def receive_handle_db_error(**kw):
+        context = kw['exception_context']
```

Comment:
But it will be `'context'` for older versions, right? Shall we fall back to this value?